### ISSUE TITLE
Add confirmation step to complete revision

### ISSUE THREAD

**Issue by mauromsl** [openlibhums/janeway#1804]
**Describe the feature**
Often times, authors complete revision requests during peer review, forgetting to upload the corrections. This can only be resolved by a staff member by clearing the `date_completed` field on the revision request record and then asking the author to resubmit.

It would be good to add a confirmation modal to the form in order to mitigate this.
We can even detect if there have been any changes to the original submission or any new files uploaded and add a warning when not the case (similar to when typesetter completes a task without uploading any galleys in typesetting plugin)

**Comment by ajrbyers** [comment]
@joemull did a confirmation get added to this view already?

**Comment by joemull** [comment]
Yes, it did get added. It checks by date uploaded, though in future it would be great to use checksums for these kind of checks.

https://github.com/BirkbeckCTP/janeway/blob/606201fb220d8b6e738241a7eb283fc71d4b1a7c/src/review/forms.py#L240-L245

**Comment by S-Haime** [comment]
Closed as this has been completed.
